wolfssl/wolfcrypt/asn.h

```diff
@@ -2161,7 +2169,7 @@ WOLFSSL_LOCAL int GetInt(mp_int* mpi, const byte* input, word32* inOutIdx,
                          word32 maxIdx);
 
 #ifdef HAVE_OID_ENCODING
-    WOLFSSL_LOCAL int EncodeObjectId(const word16* in, word32 inSz,
+    WOLFSSL_API int EncodeObjectId(const word16* in, word32 inSz,
```

Comment:
We may not need WOLFSSL_API here?

Reply:
Had to leverage it in wolfCLU

Reply:
Lets make a new facing API something like wolfSSL_EncodeObjectId (maybe not that exactly, but having either the wolfSSL_ or wc_ naming convention since public). Sanity checks on the input arguments needs reviewed once opening it up as a public facing API. This also reminds me that although in most cases people are updating wolfCLU and wolfSSL at the same time that is not always the case. Please check that wolfCLU will link to at lease a version older of wolfSSL after the changes.

Reply:
Added as a public api. Working on wolfCLU next